src/models/room.js

```diff
@@ -92,9 +92,12 @@ function synthesizeReceipt(userId, event, receiptType) {
  * "<b>detached</b>", pending messages will appear in a separate list,
  * accessbile via {@link module:models/room#getPendingEvents}. Default:
  * "chronological".
- *
  * @param {boolean} [opts.timelineSupport = false] Set to true to enable improved
  * timeline support.
+ * @param {boolean} [opts.unstableClientRelationAggregation = false]
```

Comment:
passing this down to the `EventTimelineSet` constructor will happen in a subsequent PR?

Reply:
`room.js` actually passes the whole options object through to `EventTimelineSet`, which is a bit surprising, but anyway it means it already happens:

https://github.com/matrix-org/matrix-js-sdk/blob/6078bbbe2475a1b95d211b64caa588b375c371d9/src/models/room.js#L174